### ISSUE TITLE
Feature Request: Waitlist Sort by: Current Highest Cut% minus Wanted Cut%

### ISSUE THREAD

**Issue by tfedor** [IsThereAnyDeal/IsThereAnyDeal-Issues#505]
Pretty basic feature request. I originally thought this is what the CURRENT/WANTED PRICE DIFF implied, but that must be for something completely different, like how far the price deviates from what you want (both higher or lower)

Point being I think it'd be benificial to get the wanted cut % and current cut %, and calculate how much difference (between 100 and -100) there is between those two percentages.

If this is functioning correctly, All prices listed in *green*, Id est, ones that meet the cut requirements, should appear at the top (in descending), with the exception of those that don't meet the 'Price at most' requirement.

To summarise, this method should sort the waitlist by how well the game meets (or surpasses) the user's price percentage requirement.

[Issue created by PurpleSkyHoliday: 2016-06-29]
[Last updated on bitbucket: 2017-12-23]

[Comment created by Vaevictus: 2017-12-23]
I personally don't use %, I evaluate each as a value and use that.  That said, I also want effectively the same thing, lowest price minus wanted price.  This effectively sorts them by which ones are the most prospectively valuable to me.
